src/main/java/io/vavr/jackson/datatype/serialize/VavrSerializers.java

```diff
@@ -110,7 +110,7 @@ public VavrSerializers(VavrModule.Settings settings) {
             return new LazySerializer(type);
         }
         if (Option.class.isAssignableFrom(raw)) {
-            return new OptionSerializer(type, settings.useOptionInPlainFormat());
+            return new OptionSerializer(type.getContentType(), settings.useOptionInPlainFormat());
```

Comment:
Do you think this applies to other types of Vavr? This my question does not apply to this PR, perhaps we should create new issue